task2.js

```diff
@@ -0,0 +1,16 @@
+let number = Number(prompt ("Enter your score"))
+
+ if (number > 89)
+{console.log ("you got an A")}
+
+ else if (number < 90 && number > 79)
+{console.log ("you got a B")}
+
+ else if (number < 80 && number > 69)
+    {console.log ("you got a C")}
+
+ else if (number < 70 && number > 59)
+{ console.log ("you got a D")}
+
+ else {console.log ("you got an F")}
```

Comment:
კოდის ლოგიკა სწორია, უბრალოდ სინტაქსს მიაქციე ყურადღება როგორ წერ.
მაგალითად ესე ჯობია:
```suggestion
if (number > 89) {
    console.log("you got an A")
} else if (number < 90 && number > 79) {
    console.log("you got a B")
} else if (number < 80 && number > 69) {
    console.log("you got a C")
} else if (number < 70 && number > 59) {
    console.log("you got a D")
} else {
    console.log("you got an F")
}
```